browser/admin-nl-training.js

```diff
@@ -0,0 +1,123 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of Thingpedia
+//
+// Copyright 2018 The Board of Trustees of the Leland Stanford Junior University
+//
+// Author: Giovanni Campagna <gcampagn@cs.stanford.edu>
+//
+// See COPYING for details
+"use strict";
+
+const Chart = require('chart.js');
+
+// copied from https://gist.github.com/mjackson/5311256
+function rgbToHsl(r, g, b) {
```

Comment:
surprised you are not using a library like pure-color 👍 
I will assume you get it right on this part ... lol

Reply:
I copied this code from brassau